docs/framework/deployment/deployment-guide-for-developers.md

```diff
@@ -70,8 +70,8 @@ For download links, see the section [Redistributable Packages](#redistributable-
 
```

Comment:
on line 4, please update ms.date to reflect we're making important changes

Reply:
Done.